tests/component/files/domain/models/FileMother.ts

```diff
@@ -4,18 +4,35 @@ import { DatasetVersionMother } from '../../../dataset/domain/models/DatasetMoth
 
 export class FileMother {
   static create(props?: Partial<File>): File {
+    const name = props?.name ?? faker.system.fileName()
     return {
       name: faker.system.fileName(),
       datasetVersion: DatasetVersionMother.create(),
+      citation: FileCitationMother.create(name),
       ...props
     }
   }
 
   static createRealistic(props?: Partial<File>): File {
     return this.create({
-      name: 'file.csv',
+      name: 'File Title',
       datasetVersion: DatasetVersionMother.createRealistic(),
+      citation: FileCitationMother.create('File Title'),
       ...props
     })
   }
 }
+
+export class FileCitationMother {
+  static create(fileName: string): string {
```

Comment:
I think here is where the string is composed incorrectly. (${fileName} [fileName]`)

Reply:
This is how I see the files in JSF, with the [fileName] part. If you look at the [DataCitation.java](https://github.com/IQSS/dataverse/blob/871f3735a1344fd78a506c6acdb7aa84a55f5d8c/src/main/java/edu/harvard/iq/dataverse/DataCitation.java#L218C87-L218C87) in the dataverse repo, that part is added if there is no global identifier

![image](https://github.com/IQSS/dataverse-frontend/assets/23359572/443f6ea5-da43-47d1-b76c-5c6e897d2781)


In any case, I was assuming that the citation was coming from the API, as in the Dataset page. So this `(${fileName} [fileName]`)` logic only applies to the Stoybook, then I can change it to a different string if we don't like the one with the [fileName]